dogfood/server/dogfood_server.go

```diff
@@ -38,18 +44,18 @@ var catalog = map[string]string{
 	"cow": "Grassfed",
 }
 
-type chaosDogfoodServer struct {
-	pb.UnimplementedChaosDogfoodServer
+type chaosDogfoodService struct {
```

Comment:
rename to clarify since any number of Services can be registered to a gRPC server including the new Service (disruptionlistener) which we are adding.